device-discovery/docker/Dockerfile

```
@@ -1,14 +1,22 @@
-FROM python:3.12-bullseye
-
-RUN addgroup --system netdev && useradd -m --shell /bin/bash -G netdev appuser && echo "appuser ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
+FROM python:3.12-bullseye as builder
```

Use of `as` results with warning:
```
 1 warning found (use docker --debug to expand):
 - FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 1)
```

```suggestion
FROM python:3.12-bullseye AS builder
```